src/io/flutter/view/FlutterView.java

```diff
@@ -279,9 +277,65 @@ public void actionPerformed(AnActionEvent event) {
   }
 }
 
+class RepaintRainbowAction extends AbstractToggleableAction {
+  RepaintRainbowAction(@NotNull FlutterView view) {
+    super(view, "Enable Repaint Rainbow");
+  }
+
+  protected void perform(AnActionEvent event) {
+    final Map<String, Object> params = new HashMap<>();
+    params.put("enabled", isSelected(event));
+    view.getFlutterApp().callServiceExtension("ext.flutter.repaintRainbow", params);
+  }
+}
+
+class TimeDilationAction extends AbstractToggleableAction {
+  TimeDilationAction(@NotNull FlutterView view) {
+    super(view, "Enable Slow Animations");
+  }
+
+  protected void perform(AnActionEvent event) {
+    final Map<String, Object> params = new HashMap<>();
+    params.put("timeDilation", isSelected(event) ? 5.0 : 1.0);
+    view.getFlutterApp().callServiceExtension("ext.flutter.timeDilation", params);
+  }
+}
+
+class HideSlowBannerAction extends AbstractToggleableAction {
+  HideSlowBannerAction(@NotNull FlutterView view) {
+    super(view, "Hide Slow Mode Banner");
+  }
+
+  @Override
+  protected void perform(AnActionEvent event) {
+    final Map<String, Object> params = new HashMap<>();
+    params.put("enabled", !isSelected(event));
```

Comment:
This one is reversed - all the others are off by default - the slow mode banner is on by default (but good eyes!)